DNN Platform/Library/Entities/Content/Workflow/Entities/WorkflowState.cs

```diff
@@ -20,6 +20,7 @@
 #endregion
 
 using System.ComponentModel.DataAnnotations;
+using DotNetNuke.Common.Utilities;
```

Comment:
Maybe left there after trying some stuff then removing it - this doesn't affect anything.